docs/topics/faq.md

```diff
@@ -126,3 +126,9 @@ android {
 ```
 It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/272).
 Additionally, it might be worth trying to run your tests with VisualVM to monitor memory usage and identify potential leaks.
+
+### Q: Roborazzi's Gradle task is not displayed.
+
+**A:** It is discussed in [this issue](https://github.com/takahirom/roborazzi/issues/493).  
+To enable the display of Roborazzi tasks, please enable ***Configure all Gradle tasks during Gradle Sync (this can make Gradle Sync slower)*** in the settings.  
+<img src="/docs/images/roborazzi_gradle_task_setting.png" width="800" />
```

Comment:
Thanks. This FAQ section's question is unclear about what it refers to. It might be helpful to specify that it is related to the Roborazzi IntelliJ Plugin. 👀